### DISCUSSION TITLE
Scan whole cluster efficiently with CLI mode

### DISCUSSION THREAD

**Discussion by nilesh-akhade** [aquasecurity/starboard#916] [Questions & Help] [answered]
The config audit and vulnerability plugins require workload as an argument.
Passing all workload is very tedious.
Even if we iterate over and pass all workloads to the starboard then there will be redundant scan of images (if different pods are using same container images).

My question is - How can we scan all workloads from the cluster efficiently? Is there any mode in starboard?

**Comment by danielpacak** [comment, marked as answer]
Starboard CLI (as of v0.14) was designed to scan a single workload or a single cluster node. To scan the whole cluster we recommend using Starboard Operator.
